.dotnet/src/Custom/Audio/AudioTranscription.cs

```diff
@@ -6,7 +6,7 @@ namespace OpenAI.Audio;
 public partial class AudioTranscription
 {
     // CUSTOM: Made private. This property does not add value in the context of a strongly-typed class.
-    private CreateTranscriptionResponseVerboseJsonTask Task { get; } = CreateTranscriptionResponseVerboseJsonTask.Transcribe;
+    private CreateTranscriptionResponseVerboseJsonTask Task { get; }
```

Comment:
We can undo this change. I think it was carry over from the previous PR?

Reply:
Yes, it must have survived default merge strategy. Odd.
Looking at it, we should probably do a pass of types like this, stub them as `object` or similar, and just suppress the types; it's quite a bit of extra code that just adds incremental bloat.